### ISSUE TITLE
Set assumedEvictionErrorLevel to Level.Error

### ISSUE THREAD

**Issue by jchapuis** [VirtusLab/besom#487]
There is an intrinsic eviction warning in the latest build of `besom-core` with `com.lihaoyi:sourcecode` . `pprint` depends on `0.2.7` while `besom-core` depends on `0.3.1`.

Setting `assumedEvictionErrorLevel := Level.Error` should help in surfacing these issues.

**Comment by lbialy** [comment]
Interesting. We use scala-cli to build the whole thing and it didn't catch it, apparently.

**Comment by jchapuis** [comment]
Yes so this is actually quite problematic for us as we are using sbt with strict eviction. Because of this intrinsic compatibility issue in the library, in all projects using besom we have to add `libraryDependencySchemes += "com.lihaoyi" %% "sourcecode" % "always"`

**Comment by lbialy** [comment]
Would a quickfix release of 0.3.2 with correct dependency tree help?

**Comment by lbialy** [comment]
@Gedochao are you aware if scala-cli has any way to prevent eviction-related issues like this? [Docs](https://scala-cli.virtuslab.org/docs/guides/introduction/dependencies) are silent on this topic.

**Comment by jchapuis** [comment]
@lbialy maybe I should have answered directly, yes a quickfix release would help as right now this `libraryDependencySchemes` is propagating throughout our repos 🙂

**Comment by lbialy** [comment]
For reference: the only way to validate this, for now, is to use `scala-cli export . --sbt` in `core` and then `sbt evicted` in `dest` directory. This shows possible versioning problems. I have fixed them manually in #500.

**Comment by jchapuis** [comment]
@lbialy thanks! I can confirm it works 🎉
